docs/TROUBLESHOOTING.md

```diff
@@ -102,6 +102,16 @@ ControllerMode = dual
 JustWorksRepairing = confirm
 ```
 
+#### Xbox Wireless controller
```

Comment:
Keep a double new-line in front of headlines (also for the next headline below). I know not all documentation is converted to this style but newly added content should do it.